src/cryptography/hazmat/primitives/twofactor/totp.py

```diff
@@ -31,3 +32,8 @@ def generate(self, time):
     def verify(self, totp, time):
         if not constant_time.bytes_eq(self.generate(time), totp):
             raise InvalidToken("Supplied TOTP value does not match.")
+
+    def get_provisioning_uri(self, account_name, issuer=None):
```

Comment:
In general we tend to avoid providing defaults for arguments. I think I'd prefer this just be `issuer` and then document that it's string or None. Do you have an opinion @alex?